packages/go/openapi/src/paths/asset-isolation.asset-group-tags.search.yaml

```
@@ -0,0 +1,74 @@
+  responses:
+    200:
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: object
+                properties:
+                  tags: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)